public/js/app.js

```diff
@@ -142,13 +142,16 @@ var makeComment = function(evt){
 
           if(lastComment.length < 1){
             newComment =
-              '<div><h4>1 Responses</h4>'
+              '<div><h4>1 Response</h4>'
             + newComment
             + '</div>';
             form.parent().prev().after(newComment);
+            document.getElementById("comments-project-" + respData.project_id).innerHTML = 1;
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Identifier 'project_id' is not in camel case.